src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-720.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2021-07-02'
 version: 7.2.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-7.2.0.gem'
+features: ["Configuration of expected/ignored errors by class name, status code, and message now avaliable"]
+bugs: ["Update conflicts with use of Module#Prepend", "kwarg support fixed for Rack middleware instrumentation"]
```

Comment:
Is this one supposed to have the same string as the previous version? If so, can we make either one more specific?

Reply:
I made these notes a [bit different](https://github.com/newrelic/docs-website/pull/11703/commits/f755804b3a1fea8abcd87d362cc92a4c34c3ea03#diff-c25ca68fc6fe811795b84aebe7d89c43909f63d13e149fa244d3b8b0246595aeR7)